addons/web/static/src/views/fields/statusbar/statusbar_field.js

```diff
@@ -147,10 +150,33 @@ export class StatusBarField extends Component {
         }));
     }
 
+    changeTimeState() {
+        this.state.showTimeState = !this.state.showTimeState;
+        browser.localStorage.setItem('showTimeState', this.state.showTimeState);
+    }
+
+    setTimeInStage(items) {
+        const timePerStage = JSON.parse(this.props.record.data.time_per_stage);
+        if (Object.keys(timePerStage).length) {
```

Comment:
Since this is a validation check and we don't do anything without it, you can just `return'` if `falsy`.

It avoids a large number of nested statements.